### ISSUE TITLE
Better file not found error on read_csv

### ISSUE THREAD

**Issue by jbe456** [atoti/atoti#43]
## Description

Reported by a first time user:
> the exception was not clear. When `session.read_csv` was looking for the data file, it could not find it. But it throws a pretty weird exception about `py4j` with a java error exception and atoti java exception. Took a minute to realize the error was not from the jvm but due to the incorrect file path. This is compared to the error produced by pandas when using read_csv.

Here is the error thrown by atoti's `read_csv`:
![image](https://user-images.githubusercontent.com/4415696/82180276-7cac8a00-98e0-11ea-89a5-333e05bae00d.png)

Here is the error thrown by pandas' `read_csv`:
![image](https://user-images.githubusercontent.com/4415696/82180306-8afaa600-98e0-11ea-9cc9-474b1495a7c8.png)

I see at least two difference with pandas:
1. they have a named error `FileNotFoundError`
1. atoti has two errors: "During handling of the above exception, another exception occured"

We should at least fix (2.)

**Comment by fabiencelier** [comment]
I think we used to have a single stack trace, I don't know when this changed to 2. We should try to remove this

Otherwise the only error message is "File not found XXX", I think it is clear enough to understand the issue

**Comment by rcolle** [comment]
This was fixed in 0.4.1. Here is the new error stack:

```
>>> store = session.read_csv("unknown")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\tools\miniconda3\envs\atoti\lib\site-packages\atoti\_docs_utils.py", line 18, in wrapper
    return func(*args, **kwargs)
  File "C:\tools\miniconda3\envs\atoti\lib\site-packages\atoti\session.py", line 418, in read_csv
    mode,
  File "C:\tools\miniconda3\envs\atoti\lib\site-packages\atoti\exceptions.py", line 75, in catch_py4j_exceptions
    ) from None
atoti.exceptions.AtotiJavaException: File not found C:\Users\r.colle\unknown
                    Get the full logs with : session.logs_tail(-1)
                    Get the logs path with : session.logs_path
```
